tls/datadog_checks/tls/tls.py

```diff
@@ -338,34 +332,5 @@ def local_cert_loader(self, cert):
     @property
     def tls_context(self):
```

Comment:
You can actually remove this whole `propertry` method. `self.get_tls_context()` returns always the same context instance by default.